test/Sentry.Extensions.Logging.Tests/SentryLoggerTests.cs

```diff
@@ -139,6 +155,26 @@ public void LogCritical_NotMatchingFilter_CapturesEvent()
                 .CaptureEvent(Arg.Any<SentryEvent>());
         }
 
+        [Fact]
+        public void LogCritical_NotMatchingFilter_AddsBreadcrumb()
+        {
+            var scope = new Scope();
+            _fixture.Hub.ConfigureScope(Arg.Invoke(scope));
+
+            const string expected = "message";
+            _fixture.Options.AddLogEntryFilter((_, __, ___, ____) => false);
+            _fixture.Options.AddLogEntryFilter((_, __, ___, ____) => false);
+
+            var sut = _fixture.GetSut();
+
+            sut.LogCritical(expected);
+
+            _fixture.Hub.Received(1)
+                .ConfigureScope(Arg.Any<Action<Scope>>());
```

Comment:
`l`: Maybe it makes sense to check if the hub received the right scope and not just any?

Reply:
The scope instance comes from the hub so doesn't maatter for the caller which hub